ml_inference_offloading/src/main/java/ai/nnstreamer/ml/inference/offloading/App.kt

```diff
@@ -0,0 +1,18 @@
+package ai.nnstreamer.ml.inference.offloading
+
+import android.app.Application
+import android.content.Context
+
+class App : Application() {
+    init{
+        instance = this
+    }
+
+    companion object {
+        lateinit var instance: App
+
+        fun context() : Context {
+            return instance.applicationContext
+        }
+    }
+}
```

Comment:
A minor question.
Doesn't Kotlin source leave a newline at the end?

Reply:
It depends, but in this case, it is not my intention. This is because I didn't configure my Android Studio Editor preference well as @myungjoo pointed out.